src/reporters/json.ts

```diff
@@ -285,6 +298,7 @@ export default class JSONReporter extends BaseReporter {
             url,
             status,
             metrics,
+            screenshot_ref: screenshotRef,
```

Comment:
Where does the screenshot get constructed from the individual blocks? Uptime UI?

Reply:
That's a good question, maybe there, maybe in Kibana.

Another thought is it may be worth pre-rendering small thumbnails. Assembling 20 of these for a list of thumbnails may be too much. Ideally we'd have a caching CDN

Reply:
If kibana in cloud already uses HTTP/2, We can just do individual reqs as we would yield better caching for both screenshots and filmstrips moving forward.

Reply:
Let's move this out of the `payload` field and into `synthetics.screenshot_ref`. Statistics on these fields could be useful, especially in establishing cache-hit ratio, which will be useful for debugging customer storage issues.

Reply:
Just made the change ^^